samples/KafkaFlow.Sample.ConsumerThrottling/Program.cs

```diff
@@ -68,7 +69,7 @@
                                                     .AddAction(a => a.AboveThreshold(10).ApplyDelay(1_000))
                                                     .AddAction(a => a.AboveThreshold(20).ApplyDelay(5_000))
                                                     .AddAction(a => a.AboveThreshold(30).ApplyDelay(10_000)))
-                                            .AddSerializer<JsonCoreSerializer>()
+                                            .AddDeserializer<JsonCoreDeserializer>()
```

Comment:
(Not related with this change in specific)

Since this is still in draft I imagine that the documentation part was not taken care of yet. Nevertheless, I will leave this discussion here as a reminder to update de following references in KafkaFlow documentation:

(The references that need to be changed are only references in the consumer context. Also, these are only some that I collected during the review, it may be missing some other references).

.AddSingleTypeSerializer -> .AddSingleTypeDeserializer
.AddSerializer -> .AddDeserializer
.BatchConsume -> AddBatching
.AddSchemaRegistryAvroSerializer -> .AddSchemaRegistryAvroDeserializer
.AddSchemaRegistryProtobufSerializer -> .AddSchemaRegistryProtobufDeserializer
.AddSchemaRegistryAvroSerializer -> AddSchemaRegistryAvroDeserializer
.AddCompressor -> .AddDecompressor
ProtobufNetSerializer -> ProtobufNetDeserializer
ConfluentAvroSerializer -> ConfluentAvroDeserializer
NewtonsoftJsonSerializer -> NewtonsoftJsonDeserializer
ConfluentJsonSerializer -> ConfluentJsonDeserializer
ConfluentProtobufSerializer -> ConfluentProtobufDeserializer

Reply:
@joelfoliveira should the documentation be taken care of here? https://github.com/Farfetch/kafkaflow/issues/304